internal/proxy/task.go

```diff
@@ -1286,7 +1287,10 @@ func (lct *loadCollectionTask) OnEnqueue() error {
 }
 
 func (lct *loadCollectionTask) PreExecute(ctx context.Context) error {
-	log.Debug("loadCollectionTask PreExecute", zap.String("role", typeutil.ProxyRole), zap.Int64("msgID", lct.Base.MsgID))
+	traceID, _, _ := trace.InfoFromContext(ctx)
+	log.Debug("loadCollectionTask PreExecute",
```

Comment:
log.Ctx(ctx).Debug() may be better, this makes sure all fields in context will be logged, more extensible